CHANGELOG.md

```diff
@@ -97,6 +97,7 @@ By @teoxoy [#6134](https://github.com/gfx-rs/wgpu/pull/6134).
 - Fix error message that is thrown in create_render_pass to no longer say `compute_pass`. By @matthew-wong1 [#6041](https://github.com/gfx-rs/wgpu/pull/6041)
 - Add `VideoFrame` to `ExternalImageSource` enum. By @jprochazk in [#6170](https://github.com/gfx-rs/wgpu/pull/6170)
 - Document `wgpu_hal` bounds-checking promises, and adapt `wgpu_core`'s lazy initialization logic to the slightly weaker-than-expected guarantees. By @jimblandy in [#6201](https://github.com/gfx-rs/wgpu/pull/6201)
+- Ensure safety of indirect dispatch. By @teoxoy in [#5714](https://github.com/gfx-rs/wgpu/pull/5714)
```

Comment:
question: My hot take is that most users will either bounce off of this, or get spooked. Is that desired? Did we want to elaborate on why users might care about this?

Reply:
I was thinking that interested people would check the PR directly and see:

> Ensure safety of indirect dispatch by injecting a compute shader that validates the content of the indirect buffer.

But maybe we can include that in the changelog directly?

Reply:
Yeah, I think the longer explanation is still short enough for the `CHANGELOG`. I'll edit the PR to include that. 👍🏻

Reply:
Implemented as agreed with a force-push of c851e983.